packages/ihe-gateway-sdk/src/models/patient-discovery/patient-discovery-responses.ts

```diff
@@ -4,32 +4,9 @@ import {
   baseResponseSchema,
   externalGatewayPatientSchema,
   XCPDGatewaySchema,
+  patientResourceSchema,
 } from "../shared";
 
-export const inboundPatientResourceSchema = z.object({
-  name: z
-    .array(
-      z.object({
-        family: z.string().optional(),
-        given: z.array(z.string()).optional(),
-      })
-    )
-    .optional(),
-  gender: z.enum(["male", "female", "undifferentiated"]).optional(),
```

Comment:
Why are we removing this type? The `gender` prop here is different that the one on `patientResourceSchema`

Reply:
we had no type validation before on outbound reqs so it didnt get flagged, but the fhir gender is
`'male' | 'female' | 'other' | 'unknown';`
and that doesnt line up with 
`"male", "female", "undifferentiated"`